nipype/interfaces/freesurfer/tests/test_auto_BBRegister.py

```diff
@@ -18,6 +18,7 @@ def test_BBRegister_inputs():
     usedefault=True,
     ),
     init=dict(argstr='--init-%s',
+    mandatory=True,
```

Comment:
This was added on a system with FreeSurfer v5.3. The interface changed with v6.0, and we've set the tests to follow v6.0 (see #1811 and #1817).

TLDR This change should be reverted.